packages/react-aria-components/src/Calendar.tsx

```diff
@@ -315,7 +315,12 @@ export interface CalendarCellRenderProps {
    * Whether the cell is part of an invalid selection.
    * @selector [data-invalid]
    */
-  isInvalid: boolean
+  isInvalid: boolean,
+  /**
+   * Whether the cell is today.
+   * @selector [data-today]
+   */
+  isToday: boolean
```

Comment:
figured this was a nice thing to expose, i could do it in Calendar directly though if we don't like it